schemas/dab.draft.schema.json

```diff
@@ -693,6 +693,166 @@
         }
       }
     },
+    "autoentities": {
+      "type": "object",
+      "description": "Auto-entity definitions for pattern matching",
```

Comment:
Please look at the enhancement section description titled `Suggested JSON Schema Update` for accurate description of these properties in the JSON schema. 

For autoentities, this should be `Defines automatic entity generation rules for MSSQL tables based on include/exclude patterns and defaults.`